legacy/firmware/fido2/ctap_trans.c

```
@@ -1142,15 +1142,16 @@ void ctap_hid_keepalive_register(void) {
   if (transport_type == TRANSPORT_BLE) {
     register_loop_callback(ctap_hid_keepalive_status, timer_ms(), timer1s / 12);
   } else {
-    register_timer("ctap_keepalive", timer1s / 12, ctap_hid_keepalive_status);
+    register_timer(TIMER_NAME_CTAP_KEEPALIVE, timer1s / 12,
+                   ctap_hid_keepalive_status);
   }
 }
 
 void ctap_hid_keepalive_unregister(void) {
```

Cppcheck (2.19.0)

[style] The function 'ctap_hid_keepalive_unregister' should have static linkage since it is not used outside of its translation unit.

(staticFunction)